pagespeed/controller/rpc_handler.h

```diff
@@ -189,13 +189,11 @@ void RpcHandler<AsyncService, RequestT, ResponseT>::InitDone(RefPtrT ref) {
   } else {
     // When state is FINISHED, it shouldn't be possible for anything else to be
```

Comment:
Ugh, sorry - another silly nit: You're not relying on the refcount to force a disconnect - you're explicitly forcing it. I think it would be more correct to say "When state is FINISHED, we were shutdown before we started. Propagate that as CANCELLED to any client."

Reply:
No worries, thanks for taking a good look at this!!
Amended as suggested.